win32ss/gdi/ntgdi/freetype.c

```diff
@@ -1423,14 +1438,21 @@ IntGdiLoadFontsFromMemory(PGDI_LOAD_FONT pLoadFont)
         {
             if (FT_IS_SFNT(Face))
             {
-                Length = pValueName->Length + 3 * sizeof(WCHAR) + NameLength + sizeof(UNICODE_NULL);
+                // L"... & Name StyleName\0"
+                Length = pValueName->Length + 3 * sizeof(WCHAR) + Entry->FaceName.Length +
+                         sizeof(L' ') + Entry->StyleName.Length + sizeof(UNICODE_NULL);
                 pszBuffer = ExAllocatePoolWithTag(PagedPool, Length, TAG_USTR);
                 if (pszBuffer)
                 {
                     RtlInitEmptyUnicodeString(&NewString, pszBuffer, (USHORT)Length);
                     RtlCopyUnicodeString(&NewString, pValueName);
                     RtlAppendUnicodeToString(&NewString, L" & ");
                     RtlAppendUnicodeStringToString(&NewString, &Entry->FaceName);
+                    if (Entry->StyleName.Length > 0)
+                    {
+                        RtlAppendUnicodeToString(pValueName, L" ");
```

Comment:
I think you mean `&NewString` here, not `pValueName`?

Reply:
Yes.